File: SampleSetup/Protocol/Body/ApiBodyWithNested.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public class ApiBodyWithNested
    {
        public string Prop1 { get; set; } = "";
        public int? Prop2 { get; set; } = null;

        public class Nested_1
        {
            public string Prop_1_1 { get; set; } = "";
            public int? Prop_1_2 { get; set; } = null;

            public class Nested_1_2
            {
                public string Prop_1_2_1 { get; set; } = "";
                public int? Prop_1_2_2 { get; set; } = null;
            }
        }
    }
}

File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Body/ApiBodyWithNested.ts
export class ApiBodyWithNested {
    Prop1: string = '';
    Prop2: number | null = null;

    export class Nested_1 {
        Prop_1_1: string = '';
        Prop_1_2: number | null = null;

        export class Nested_1_2 {
            Prop_1_2_1: string = '';
            Prop_1_2_2: number | null = null;
        }
    }
}
